### ISSUE TITLE
RCMeetings Pkg Change

### ISSUE THREAD

**Issue by TSPARR** [autopkg/homebysix-recipes#302]
Looks like RingCentral made some changes to their pkg installer. From an initial review, I believe this removes the need for repackaging it altogether. Wanted to get another set of eyes on it before PR'ing.

**Comment by TSPARR** [comment]
I started down the path on this a little bit. I believe the .download doesn't need to change at all, but the .pkg would use the following processors:
```<dict>
        <key>Arguments</key>
        <dict>
          <key>destination_path</key>
          <string>%RECIPE_CACHE_DIR%/downloads/unpack</string>
          <key>flat_pkg_path</key>
          <string>%pathname%</string>
        </dict>
        <key>Processor</key>
        <string>FlatPkgUnpacker</string>
      </dict>
      <dict>
        <key>Arguments</key>
        <dict>
          <key>destination_path</key>
          <string>%RECIPE_CACHE_DIR%/downloads/payload</string>
          <key>pkg_payload_path</key>
          <string>%RECIPE_CACHE_DIR%/downloads/unpack/install.pkg/Payload</string>
        </dict>
        <key>Processor</key>
        <string>PkgPayloadUnpacker</string>
      </dict>
      <dict>
        <key>Arguments</key>
        <dict>
          <key>input_plist_path</key>
          <string>%RECIPE_CACHE_DIR%/downloads/payload/RingCentral Meetings.app/Contents/Info.plist</string>
          <key>plist_version_key</key>
          <string>CFBundleShortVersionString</string>
        </dict>
        <key>Processor</key>
        <string>Versioner</string>
      </dict>
      <dict>
        <key>Processor</key>
        <string>PkgCopier</string>
        <key>Arguments</key>
        <dict>
          <key>source_pkg</key>
          <string>%pathname%</string>
          <key>pkg_path</key>
          <string>%RECIPE_CACHE_DIR%/%NAME%_%version%.pkg</string>
        </dict>
      </dict>```

For some reason, the .munki imports that as "RingCentral Meetings_19.4.152079.0420-1.0" and I haven't had the time to dig deep enough into why that is just yet. Maybe someone else will be able to take it and run with it from here before I do.

**Comment by TSPARR** [comment]
This was resolved by #303
